pkg/history/populate_headers.go

```diff
@@ -61,3 +62,7 @@ func RetrieveAndUpdateHeaders(blockChain core.BlockChain, headerRepository datas
 	}
 	return len(blockNumbers), nil
 }
+
+func getBlockNumbersString(blockNumbers []int64) string {
```

Comment:
This looks the same as `getBlockRangeString` and they're in the same package so should be able to reuse one or the other without having to export the function.